libs/NfcCoreLib/lib/LibNfc/phLibNfc_Se.c

```diff
@@ -1329,6 +1383,73 @@ static NFCSTATUS phLibNfc_SetSeModeSeqComplete (void* pContext,NFCSTATUS Status,
     return wStatus;
 }
 
+static NFCSTATUS
+phLibNfc_SePowerAndLinkCtrlSeq(void *pContext,
+                               NFCSTATUS wStatus,
+                               void *pInfo)
+{
+    NFCSTATUS wIntStatus;
+    pphLibNfc_LibContext_t pLibContext = (pphLibNfc_LibContext_t)pContext;
+    UNUSED(pInfo);
+    UNUSED(wStatus);
+    PH_LOG_LIBNFC_FUNC_ENTRY();
+    if ((NULL != pLibContext) && (phLibNfc_GetContext() == pLibContext))
+    {
+        wIntStatus = phNciNfc_Nfcee_SePowerAndLinkCtrlSet(pLibContext->sHwReference.pNciHandle,
+                                (void *)pLibContext->sSeContext.pActiveSeInfo->hSecureElement,
+                                pLibContext->sSeContext.ePowerLinkMode,
+                                (pphNciNfc_IfNotificationCb_t)&phLibNfc_InternalSequence,
+                                (void *)pLibContext);
+    }
+    else
+    {
+        PH_LOG_LIBNFC_CRIT_STR("Invalid Context passed from lower layer!");
```

Comment:
Assign 'wIntStatus' an error code here?

Reply:
I have assigned wIntStatus at declaration to NFCSTATUS_FAILED.